project/threads/templates/threads/base/links.html

```diff
@@ -10,14 +10,12 @@
 <link rel="shortcut icon" href="/favicon.ico">
 <meta name="msapplication-TileImage" content="/mstile-144x144.png">
 <meta name="msapplication-TileColor" content="#fcffff">
-
-<link type="text/css" rel="stylesheet" href="{% static "dependencies/materialize.min.css" %}">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
 <link type="text/css" rel="stylesheet" href="{% static "dependencies/magicsuggest-min.css" %}">
 <script type="text/javascript" src="{% static "dependencies/jquery.min.js" %}"></script>
 <script type="text/javascript" src="{% static "dependencies/underscore.js" %}"></script>
 <script type="text/javascript" src="{% static "dependencies/backbone-min.js" %}"></script>
-<script type="text/javascript" src="{% static "dependencies/materialize.min.js" %}"></script>
 <script type="text/javascript" src="{% static "dependencies/magicsuggest-min.js" %}"></script>
 <script type="text/javascript" src="{% static "js/base.js" %}"></script>
-<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script> 
```

Comment:
Please pay *careful* attention to the work.

We don't want to delete the Material Icons font as part of this pull request.

```suggestion
<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script> 
```